test/start_server_test.js

```diff
@@ -51,14 +51,17 @@ suite('startServer', () => {
       .end(done);
   });
 
-  test('404s .html files', (done) => {
-    let app = getApp({
-      root: __dirname,
-    });
-    supertest(app)
-      .get('/foo.html')
+  ['html', 'js', 'json', 'css', 'png', 'jpg', 'jpeg', 'gif'].forEach((ext) => {
```

Comment:
love using this list as a test dataset though! If we can make the above more flexible, you could remove a few of these and/or add in some less common ones.